src/index.html

```diff
@@ -5,8 +5,367 @@
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__starsReviews">Reviews: 5</p>
+          </section>
+
+          <div class="card__price card__price--margin">
+            <p class="card__price-label">Price:</p>
+            <p class="card__price-value">&dollar;2,199</p>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+            class="card__image"
+          >
+
+          <h1  class="card__product">
+            APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+
+          <p class="card__productCode">
+            Product code: 195434
+          </p>
+
+          <section class="card__stars card__stars--margin">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__starsReviews">Reviews: 5</p>
+          </section>
+
+          <div class="card__price card__price--margin">
+            <p class="card__price-label">Price:</p>
+            <p class="card__price-value">&dollar;2,199</p>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+
+
+        <div class="card">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+            class="card__image"
+          >
+
+          <h1  class="card__product">
+            APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+
+          <p class="card__productCode">
+            Product code: 195434
+          </p>
+
+          <section class="card__stars card__stars--margin">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__starsReviews">Reviews: 5</p>
+          </section>
+
+          <div class="card__price card__price--margin">
+            <p class="card__price-label">Price:</p>
+            <p class="card__price-value">&dollar;2,199</p>
+          </div>
+          <a
```

Comment:
Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements. Please, fix it in all places.